src/page-multi-facility/CreateBaselineScenarioPage.tsx

```diff
@@ -47,11 +47,29 @@ const CreateBaselineScenarioPage: React.FC = () => {
       <ModalDialog open title="Welcome">
         <ModalContent>
           <WelcomeText>
-            Welcome to your new scenario. To get started, add in facility data
-            on the right-hand side of the page. Your initial scenario is also
-            your 'Baseline' - meaning this is where you should keep real-world
-            numbers about the current state of your facilities, their cases, and
-            mitigation steps.
+            This tool will help you model Covid-19 cases in a set of criminal
+            justice facilities (jails or prisons), and provide a forecast of
+            likely cases given your facility policies and populations. For more
+            information and for a list of the corrections and epidemiology
+            collaborators who made this model possible, see
+            <a href="https://www.recidiviz.org/collaborators"> recidiviz.org/collaborators</a>
+            .
+            <br />
+            <br />
+            To begin, we've created a 'Baseline Scenario' for you, where you can
+            add your current facilities. Think of a 'Scenario' as you would a
+            folder - it's just a collection of facilities that are being
+            modeled. The 'Baseline' scenario is one you should keep as close to
+            the real-world as possible. When you want to model something new (a
+            policy change, for instance), you'll be able to make a copy of your
+            baseline scenario, make the change there, and compare it to
+            baseline. (Note: Creating multiple scenarios isn't ready just yet,
+            but it will be available within the next few days.)
+            <br />
+            <br />
+            If you have any questions or concerns, feel free to drop us a note
+            at
+            <a href="mailto:covid@recidiviz.org"> covid@recidiviz.org</a>.
```

Comment:
Nit: Leading space after the opening `a` tag

Reply:
That was by design to ensure there was a space between at and the email address, but I think I know a better way to do it.